specs/deneb/validator.md

```diff
@@ -44,7 +44,8 @@ Note: This API is *unstable*. `get_blobs_and_kzg_commitments` and `get_payload`
 Implementers may also retrieve blobs individually per transaction.
```

Comment:
Ah and I checked, the engine api doesn't currently give you the proofs. so right now, adding the proofs to this would be not in sync with the engine api 

That said abstracting it to magically show up in the validator spec does seem correct/safe so I'm okay leaving this as is

Reply:
We should change the engine to give the proofs, because they should be part of the `BlobTransactionNetworkWrapper`.

Which made me noticed, it seems the EIP has not been updated to include this yet. Seems to be included in this PR: https://github.com/ethereum/EIPs/pull/6610/

Reply:
I see, let's go to engine api after eip is merged